framework-cloud/framework-openshift-templates/src/main/java/org/kie/cloud/openshift/scenario/ClusteredWorkbenchRuntimeSmartRouterTwoKieServersTwoDatabasesScenarioImpl.java

```diff
@@ -132,12 +123,6 @@ protected void deployKieDeployments() {
         logger.info("Waiting for Kie servers and Smart router to register itself to the Workbench.");
         KieServerControllerClientProvider.waitForServerTemplateCreation(workbenchRuntimeDeployment, 3);
 
-        // Restart Smart router due to problems with Monitoring console clustering, should be fixed for 7.5.
```

Comment:
@Sgitario can you please check that this removed do not break tests. If I remember correctly this should be related with this [issue](https://issues.redhat.com/browse/BAPL-1267)

Reply:
Sure, I will. I'm running all the suite. However, no test is working now because the changes in https://issues.redhat.com/browse/RHPAM-2175.

I will check all the results afterwards and provide more details to this PR before merging.